libraries/third_party/recaptchalib.php

```diff
@@ -1,277 +1,32 @@
 <?php
-/*
```

Comment:
I removed the ReCaptcha copyright because this file no longer contains functions from their library.